### ISSUE TITLE
Lots of errors on ubuntu 15.10 server. Not sure if I'm a noob or if its incompatible with something. 

### ISSUE THREAD

**Issue by innominata** [Bisa/factorio-init#52]
Factorio launches in the background, but factorio-init doesn't recognise it. To stop it I have to 'killall factorio'. Is it an issue with ubuntu or me?

```
factorio@orange:/opt/factorio/mods$ factorio start
DEBUG LOG: could not find a pid for binary: "/opt/factorio/bin/x64/factorio"
DEBUG LOG: last modified save: /opt/factorio/saves/factorio-init-save.zip
DEBUG LOG: using existing factorio-init-save.zip
DEBUG LOG: could not find a pid for binary: "/opt/factorio/bin/x64/factorio"
DEBUG LOG: could not find a pid for binary: "/opt/factorio/bin/x64/factorio"
Still not running, waiting a while longer...
DEBUG LOG: could not find a pid for binary: "/opt/factorio/bin/x64/factorio"
DEBUG LOG: could not find a pid for binary: "/opt/factorio/bin/x64/factorio"
DEBUG LOG: could not find a pid for binary: "/opt/factorio/bin/x64/factorio"
DEBUG LOG: could not find a pid for binary: "/opt/factorio/bin/x64/factorio"
DEBUG LOG: could not find a pid for binary: "/opt/factorio/bin/x64/factorio"
DEBUG LOG: could not find a pid for binary: "/opt/factorio/bin/x64/factorio"
DEBUG LOG: could not find a pid for binary: "/opt/factorio/bin/x64/factorio"
DEBUG LOG: could not find a pid for binary: "/opt/factorio/bin/x64/factorio"
Failed to start, aborting.
```

**Comment by Bisa** [comment]
not sure, try pulling from master and re-check the config.example to make sure you did not miss any options

**Comment by Bisa** [comment]
closing this for now, please see possible relation/fix to #53
